index.html

```diff
@@ -1,47 +1,199 @@
+            their robot in Python to compete in the competition.
+          </p>
+        </div>
+        <div class="column center-label s-4-12">
+          <img src="{{ '/images/advertising/team-building-robot.jpg' | prepend: site.baseurl }}"
+            alt="Students and a teacher sitting around their robot" />
+          <h3>Building a robot</h3>
+          <p>
+            There's nothing quite like building something and bringing it to
+            life. Using our kit, teams will build a fully autonomous robot. The
+            best teams pour their imagination into the design and the look of
+            their robots.
+          </p>
+        </div>
       </div>
     </div>
   </div>
-  <div class="fixed-width">
-    <div class="row">
-      <div class="sidebar column m-4-12">
-        <div class="card">
-          <h3 class="card-heading">Upcoming Events</h3>
+</div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Testimonials</h2>
+      <div class="row vertically-centered-row">
+        <div class="column m-9-12">
+          <blockquote class="text-italic">
+            Student Robotics provided me with the invaluable experience of working on a engineering project with a team.
+            I learned not only good engineering principals but the importance of being able to clearly communicate my
+            thoughts and reasoning.
+            Competing in Student Robotics opened the door to me directly entering the world of work as a Software
+            Engineer after completing Sixth Form.
+          </blockquote>
+          <p class="text-heavy text-center">— James Seden Smith</p>
+          <h5 class="text-center">
+            Student Robotics 2015 and 2016<br>
+            Team CLY
+          </h5>
+        </div>
+        <div class="column m-3-12 horizontally-centered-column">
+          <img class="testomonial-avatar"
+            src="{{ '/images/advertising/headshots/james-seden-smith.jpg' | prepend: site.baseurl }}"
+            alt="Headshot of James" />
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel panel-highlight panel-thin">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center top-spacing">Of competitors surveyed from the last 2 years</h2>
+      <div class="row">
+        <div class="column m-4-12">
+          <p class="statistic">100%</p>
+          <p class="statistic-label">would recommend Student Robotics</p>
+        </div>
+        <div class="column m-4-12">
+          <p class="statistic">97%</p>
+          <p class="statistic-label">plan to study a STEM degree at university</p>
+        </div>
+        <div class="column m-4-12">
+          <p class="statistic">88%</p>
+          <p class="statistic-label">are now highly interested in a STEM career</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel">
```

Comment:
Does it make sense to put the code for the sponsors box into a separate file, so it can be easily brought into any page?

Reply:
I feel like if we wanted to show sponsors on other pages we may not want them displayed in the same way as this page (we may want them in a side panel, or in a different orientation for example). I tend to prefer pulling stuff out into a shared file when we use it in another place, as opposed to pre-emptively splitting it out.